src/client/gui/gui_cmd.cpp

```diff
@@ -165,6 +165,12 @@ void cmd::GuiCmd::create_actions()
     QObject::connect(&petenv_start_action, &QAction::triggered, [this] {
         future_synchronizer.addFuture(QtConcurrent::run(this, &GuiCmd::start_instance_for, current_petenv_name));
     });
+    QObject::connect(&petenv_disable_action, &QAction::triggered,
+                     [this]
+                     {
+                         current_petenv_name = "";
+                         MP_SETTINGS.set(petenv_key, QString::fromStdString(current_petenv_name));
+                     });
```

Comment:
I am not sure if this was leftover from earlier, but I don't think we need a button to disable primary.